tasks.py

```diff
@@ -115,7 +124,11 @@ def install(args):
         cmd = 'pip install .[docs,tests]'
     else:
         cmd = 'pip install -e .[tests]'
-    run(cmd)
+
+    env = os.environ.copy()
+    if args.group != 'labextensions':
```

Comment:
Is this logic correct for when the group is 'all'?

Reply:
Right, fixed in https://github.com/jupyter/nbgrader/pull/1588/commits/ca01cd66c7664fcf254cccbd3c67e5cdac3f466d